app/models/main.py

```diff
@@ -145,6 +151,21 @@ def serialize(self):
             'applicationCloseDate': (
                 self.application_close_date and self.application_close_date.strftime(DATETIME_FORMAT)
             ),
+            'applicationsCloseAtUTC': (
```

Comment:
I'm not sure it makes sense to expose this if it won't actually have been filled out yet?

Reply:
I've got a script to inject the dates when this PR gets merged. If I don't add these to the serialization now, I'll have to do a separate PR to do it later wouldn't I? Seems as easy to add it now as later.

Reply:
Are we talking theoretically or practically? The only thing that will be posting to the `AtUTC` variant before I tear out the old one will be my script to insert the dates. Given that I plan to rip the old one out quickly, I don't think it adds much ambiguity in reality.

Also the script just makes 20 or so posts to the API, so it will take like a few seconds. Don't think we need to worry about the impact there.

Reply:
I think I'm just saying that tearing out the old field could be a PR with a migration only, and all the code changes to ignore that field could potentially be here.

And I realised after I pressed send that this table is probably the least likely one to need to worry about performance impact :-)